includes/GlobalFunctions.php

```diff
@@ -171,18 +171,19 @@ function &smwfGetSparqlDatabase() {
  * Compatibility helper for using Linker methods.
  * MW 1.16 has a Linker with non-static methods,
  * where in MW 1.19 they are static, and a DummyLinker
- * class is introduced, which can be instantaited for
- * compat reasons.
+ * class is introduced, which can be instantiated for
+ * compat reasons. As of MW 1.28, DummyLinker is being
+ * deprecated, so always use Linker.
```

Comment:
Basically....I'm not far how back MW versions you want this doc comment to be about. Since it used to return DummyLinker, I added a comment explaining why it no longer does.